src/NJsonSchema/Generation/JsonSchemaGeneratorSettings.cs

```diff
@@ -66,6 +66,9 @@ public JsonSchemaGeneratorSettings()
         /// <summary>Gets or sets a value indicating whether to flatten the inheritance hierarchy instead of using allOf to describe inheritance (default: false).</summary>
         public bool FlattenInheritanceHierarchy { get; set; }
 
+        /// <summary>Gets or sets a value indicating whether to remove duplicates.</summary>
+        public bool IgnoreInhertianceDuplicates { get; set; }
```

Comment:
Not needed with the other chnage

Reply:
Removed property as new implementation works much better